src/utils/matchers/blacklist_patterns.ts

```diff
@@ -0,0 +1,47 @@
+import { GITHUB_REPOSITORY_CONFIG_FILE_PATTERN } from "../config";
+import { MATCH_OPTIONS } from "../config";
```

Comment:
Codacy found an issue: [Multiple imports from '../config' can be combined into one.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6318502)